sdk/loadtesting/load-testing-rest/src/getTestRunCompletionPoller.ts

```diff
@@ -5,13 +5,13 @@ import type { AbortSignalLike } from "@azure/abort-controller";
 import { AbortError } from "@azure/abort-controller";
 import type { CancelOnProgress, OperationState, SimplePollerLike } from "@azure/core-lro";
 import type { TestRunCompletionPoller, PolledOperationOptions } from "./models";
-import type { AzureLoadTestingClient } from "./clientDefinitions";
+import type { AzureLoadTestingClient } from "../generated/clientDefinitions";
```

Comment:
maybe not related, but why we have `import type` here?

Reply:
this is lint:fix change